ts/src/btcmarkets.ts

```diff
@@ -155,16 +155,18 @@ export default class btcmarkets extends Exchange {
             },
             'precisionMode': TICK_SIZE,
             'exceptions': {
-                '3': InvalidOrder,
-                '6': DDoSProtection,
-                'InsufficientFund': InsufficientFunds,
-                'InvalidPrice': InvalidOrder,
-                'InvalidAmount': InvalidOrder,
-                'MissingArgument': InvalidOrder,
-                'OrderAlreadyCancelled': InvalidOrder,
-                'OrderNotFound': OrderNotFound,
-                'OrderStatusIsFinal': InvalidOrder,
-                'InvalidPaginationParameter': BadRequest,
+                'exact': {
```

Comment:
should we remove the '3' and '6' exceptions?

Reply:
yes
this is old errors from api v2, the current api v3 does not have them
you can check all imposible errors there:
https://docs.btcmarkets.net/#tag/ErrorCodes